project/MicroServiceBuild.scala

```diff
@@ -11,18 +11,17 @@ private object AppDependencies {
   import play.sbt.PlayImport._
   import play.core.PlayVersion
 
-  private val bootstrapPlayVersion = "4.6.0"
+  private val bootstrapPlayVersion = "4.8.0"
   private val domainVersion = "5.3.0"
   private val hmrcTestVersion = "3.3.0"
```

Comment:
could we update hmrctest to "3.4.0-play-25"?

Reply:
Sorted - good catch